docs/content/v2.12/admin/yb-admin.md

```diff
@@ -1618,6 +1469,159 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
 
 ---
 
+### xCluster Replication commands
+
+#### setup_universe_replication
+
+Sets up the universe replication for the specified source universe. Use this command only if no tables have been configured for replication. If tables are already configured for replication, use [alter_universe_replication](#alter-universe-replication) to add more tables.
+
+To verify if any tables are already configured for replication, use [list_cdc_streams](#list-cdc-streams).
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <target_master_addresses> \
+    setup_universe_replication \
+    <source_universe_uuid>_<replication_name> \
+    <source_master_addresses> \
+    <comma_separated_list_of_table_ids> \
+    [ <comma_separated_list_of_producer_bootstrap_ids> ]
+```
+
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
+* *source_universe_uuid*: The UUID of the source universe.
+* *replication_name*: The name for the replication.
+* *source_master_addresses*: Comma-separated list of the source master addresses.
+* *comma_separated_list_of_table_ids*: Comma-separated list of source universe table identifiers (`table_id`).
+* *comma_separated_list_of_producer_bootstrap_ids*: Comma-separated list of source universe bootstrap identifiers (`bootstrap_id`). Obtain these with [bootstrap_cdc_producer](#bootstrap-cdc-producer-comma-separated-list-of-table-ids), using a comma-separated list of source universe table IDs.
+
+{{< warning title="Important" >}}
+Enter the source universe bootstrap_ids in the same order as their corresponding table_ids.
```

Comment:
Suggestion: 
Enter the source universe bootstrap IDs in the same order as their corresponding table IDs.